lib/telemetry.js

```diff
@@ -60,5 +67,37 @@ export const startPingLoop = () => {
     point.tag('arch', arch())
     point.tag('deployment_type', DEPLOYMENT_TYPE)
     writeClient.writePoint(point)
-  }, 10 * 60 * 1000)
+    writeClient.flush().catch(handleFlushError)
+    await timers.setTimeout(10 * 60 * 1000) // 10 minutes
+  }
+}
+
+export const runMachinesLoop = async () => {
+  while (true) {
+    const point = new Point('machine')
+    point.stringField('process_uuid', processUUID)
```

Comment:
Can you please add `station_id` to these data points too? (Alternatively, remove `process_uuid` and use `station_id` only.)